guide/additional-info/changes-in-v12.md

```diff
@@ -4,7 +4,7 @@ After a long time in development, Discord.js v12 is nearing a stable release, me
 
 ## Before You Start
 
-v12 requires Node 10.2.x or higher to  use, so make sure you're up-to-date.  To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it!  There are many resources online to help you get up-to-date.
+v12 requires Node 12.x or higher to  use, so make sure you're up-to-date.  To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it!  There are many resources online to help you get up-to-date.
 
 For now, you do need Git installed and added to your PATH environment, so ensure that's done as well - again, guides are available online for a wide variety of operating systems.  Once you have Node up-to-date and Git installed, you can install v12 by running `npm install discordjs/discord.js` in your terminal or command prompt for text-only use, or `npm install discordjs/discord.js @discordjs/opus` for voice support.
```

Comment:
How about stating this way to install: `npm install discord.js`?

Reply:
Yeah that's not a bad idea, seems silly to make another PR just for that. Changed and will update description+title